data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,785 @@
+	"flare sprite" effect/scplasma/tiny
+		"frame rate" 0.6
+	"flare sound" "scplasma tiny"
+	description `In order to power the small Halite frame, the Gegno Scin developed this smaller engine system. Based on their larger plasma thrusters, these engines are much more efficient than the Vi's larger, more sluggish Ion Engines, which would never be compact or light enough to run on a fighter.`
+
+
+outfit "SC-12 Plasma Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 98000
+	mass 25
+	thumbnail outfit/smallscplasma
+	"engine capacity" -25
+	"outfit space" -25
+	thrust 10.6
+	"thrusting energy" 0.8
+	"thrusting heat" 2.9
+	"flare sprite" effect/scplasma/small
+		"frame rate" 0.5
+	"flare sound" "scplasma small"
+	description `Rather than developing on the Ion Torch design of the Gegno Vi, the Gegno Scin began work on a plasma-based thruster. They are more compact, weigh less, and produce significantly more thrust. However, since Ion Torches expel some of their heat in their thrust, the more stable and powerful SC thrusters tend to run hotter.`
+
+
+outfit "SC-12 Plasma Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 95800
+	mass 19
+	thumbnail outfit/smallscplasmasteer
+	"engine capacity" -19
+	"outfit space" -19
+	turn 270
+	"turning energy" 0.55
+	"turning heat" 1.8
+	"steering flare sprite" effect/scplasma/small
+		"frame rate" 0.5
+	"steering flare sound" "scplasma small"
+	description `The new SC plasma thrusters allow Gegno Scin ships to outmaneuver their opposing brothers. What might take a Gegno Vi ship a long time takes only a moment's worth on a Scin ship.`
+
+
+outfit "SC-14 Plasma Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 162000
+	mass 45
+	thumbnail outfit/medscplasma
+	"engine capacity" -45
+	"outfit space" -45
+	thrust 26.8
+	"thrusting energy" 2.8
+	"thrusting heat" 7.9
+	"flare sprite" effect/scplasma/med
+		"frame rate" 0.4
+	"flare sound" "scplasma medium"
+	description `SC-14 class Thrusters power the Scin's average gunboat or transport, giving their ships reliable speed. They can quickly propel any medium-sized ship forward with reasonable efficiency.`
+
+
+outfit "SC-14 Plasma Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 156400
+	thumbnail outfit/medscplasmasteer
+	"engine capacity" -34
+	mass 34
+	"outfit space" -34
+	turn 690
+	"turning energy" 1.5
+	"turning heat" 4.78
+	"steering flare sprite" effect/scplasma/med
+		"frame rate" 0.4
+	"steering flare sound" "scplasma medium"
+	description `Larger than SC-12 thrusters, SC-14 steering thrusters are one of the most powerful steering systems in Gegno space, allowing the Scin to reposition even their largest ships many times faster than their Vi counterparts.`
+
+
+outfit "SC-15 Plasma Thrusters"
+	plural "SC-15 Plasma Thrusters"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+		"Scin Architect"
```

Comment:
Same logic as for the other one, except there are 2 here.

Reply:
Same as the above, lore with the Scin. One is more weaponry focused, the other more technical. 

Hoplologist: Someone who is an [expert](https://en.wiktionary.org/wiki/expert) in [weapons](https://en.wiktionary.org/wiki/weapon) and [armour](https://en.wiktionary.org/wiki/armour) (especially [military](https://en.wiktionary.org/wiki/military) uses or knowledge thereof).

Architect (computing): A person who designs hardware, software, or networking applications and services of a specified type for a business or other organization.